R/real_vector.R

```diff
@@ -353,10 +353,18 @@ inspect_probability_vector <- function(
     return(res)
   }
 
+  value_all <- value
   value <- value[!is.na(value)]
 
   if (!(all(value >= 0) && all(value <= 1))) {
-    return(failure("Some values were outside the allowed range"))
+    mask <- value_all[value_all < 0 | value_all > 1]
```

Comment:
This is not a mask. A mask is an array with TRUE and FALSE. what you have in the [] is the mask. This is the result of the extraction of the values complying with the mask.